pkgs/applications/networking/nordvpn/default.nix

```diff
@@ -0,0 +1,82 @@
+{ autoPatchelfHook, buildFHSEnv, dpkg, fetchurl, lib, stdenv, sysctl
+, iptables, iproute2, procps, cacert, libxml2, libidn2, zlib, wireguard-tools }:
+
+let
+  pname = "nordvpn";
+  version = "3.15.5";
+
+  nordVPNBase = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = fetchurl {
```

Comment:
Is the no source code available (I assume no due to the `unfree` license) we can build from instead?

Reply:
no, unfortunately.

Reply:
I stumbled upon what I think is the NordVPN Linux client source at https://github.com/NordSecurity/nordvpn-linux.

Reply:
@solreynolds  :O you're right. How could I miss that? Well, I'll try my hand at building the package, probably this week or next weekend...